src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variants/sw-product-variants-delivery/sw-product-variants-delivery-order/index.js

```diff
@@ -25,8 +25,6 @@ Component.register('sw-product-variants-delivery-order', {
         };
     },
 
-    watch: {},
```

Comment:
it was watching you 😉

Reply:
As you mention it. I can see it now as well `:{}`

Reply:
I hope your marketing/PR department is gonna like it as much as I do :upside_down_face: :fearful: 
![In Germany, Schöppingen Code watches you!](https://user-images.githubusercontent.com/1133593/92303603-bfb96880-ef76-11ea-9b2b-2ba88c3b7964.png)